browser/resources/settings/brave_sync_page/brave_sync_setup.html

```diff
@@ -52,11 +52,12 @@ <h1>$i18n{braveSyncSetupTitle}</h1>
   </div>
 </div>
 
-<template is="dom-if" if="[[syncCodeDialogType_]]" restamp>
+<template is="dom-if" if="[[syncCodeDialogType]]" restamp>
   <settings-brave-sync-code-dialog
-    code-type="{{syncCodeDialogType_}}"
+    code-type="{{syncCodeDialogType}}"
     sync-code="{{syncCode}}"
     sync-code-validation-error="{{syncCodeValidationError_}}"
     on-done="handleSyncCodeDialogDone_"
+    on-show-get-code="handleShowGetCode_"
```

Comment:
This attribute / event handler doesn't seem used anymore? I can't find a function named `handleShowGetCode_`

Reply:
yes, this was a leftover which I forgot to remove after unsuccessful attempt to invoke code dialog from other component. 

thanks for notice, removed at https://github.com/brave/brave-core/pull/16117/commits/b1b60b413f9c4e01c3b0ffd4f44333b0393b0ffd